$NetBSD: patch-Makefile,v 1.1 2019/03/11 16:27:03 kamil Exp $

Add fallback files with functions missing on !OpenBSD.
Specify -D_OPENBSD_SOURCE to enable OpenBSD APIs on NetBSD.

--- Makefile.orig	2019-02-21 15:41:00.000000000 +0000
+++ Makefile
@@ -12,12 +12,15 @@ OBJS	 = blocks.o \
 	   md4.o \
 	   mkpath.o \
 	   mktemp.o \
+	   pledge.o \
+	   recallocarray.o \
 	   receiver.o \
 	   sender.o \
 	   server.o \
 	   session.o \
 	   socket.o \
 	   symlinks.o \
+	   unveil.o \
 	   uploader.o
 ALLOBJS	 = $(OBJS) \
 	   main.o
@@ -27,7 +30,7 @@ MANDIR	 = $(PREFIX)/man
 BINDIR	 = $(PREFIX)/bin
 
 # The -O0 is to help with debugging coredumps.
-CFLAGS	+= -O0 -g -W -Wall -Wextra -Wno-unused-parameter
+CFLAGS	+= -O0 -g -W -Wall -Wextra -Wno-unused-parameter -D_OPENBSD_SOURCE
 
 all: openrsync
 
